packages/bitcore-lib/test/transaction/transaction.js

```diff
@@ -411,7 +411,7 @@ describe('Transaction', function() {
         .change(changeAddress)
         .sign(privateKey);
       transaction.outputs.length.should.equal(2);
-      transaction.outputs[1].satoshis.should.equal(477100);
+      transaction.outputs[1].satoshis.should.equal(473500);
```

Comment:
This changes are because the Safety Margin was decreased (for the fee), hence, the output is different